overlord/cmdstate/cmdmgr.go

```diff
@@ -41,6 +41,10 @@ func Manager(st *state.State) *CommandManager {
 	return &CommandManager{runner: runner}
 }
 
+func (m *CommandManager) KnownTaskKinds() []string {
+	return m.runner.KnownTaskKinds()
```

Comment:
Can we please have a test for each of those, for the real ones (not mock/fake)?

It sounds silly, but a single one of those that happens to return []string{} will end up with a hard-to-catch bug, where tasks will magically disappear in real deployments.

Let's please double check that all the real ones are indeed returning the right thing.